rest/admin_api_auth_test.go

```diff
@@ -15,7 +16,14 @@ import (
 )
 
 func MakeUser(t *testing.T, serverURL, username, password string, roles []string) {
-	httpClient := http.DefaultClient
+	// TODO: Make work with management HTTP client on Jenkins
+	httpClient := &http.Client{
```

Comment:
Just to humor me can you try to use the management HTTP client locally and breakpoint it. See if it has the InsecureSkipVerify set?

Reply:
Had a look at this. It does not set `InsecureSkipVerify` and it does fail locally as well when this is not set. 

Looks like gocbcore would need to add this on their end on creation as you can't set it on an existing HTTP client.

Reply:
Fixed through commit b43cb76